.github/workflows/tests.yml

```diff
@@ -73,6 +73,8 @@ jobs:
     - name: Add config
       run: |
         echo "[gallia]" > gallia.toml
+        echo "verbosity = 10" >> gallia.toml
```

Comment:
verbosity of value 10 has no special meaning: edef9290411819fb7cc4d2bea237db58b73c9909. The default verbosity is INFO, then you could switch to DEBUG, that would be value `1` IIRC.

Reply:
The numbers mason, what do they mean!

Let it be 1 then...